pom.xml

```diff
@@ -53,10 +49,9 @@
         <maven.compiler.version>3.8.1</maven.compiler.version>
         <project.java.version>1.11</project.java.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <unpack.includes>org/kuali/rice/**/*.class,org/kuali/rice/**/*.properties,META-INF/**,org/kuali/kfs/kim/impl/role/*.xml</unpack.includes>
+        <unpack.includes>META-INF/**,org/kuali/kfs/kim/impl/role/*.xml</unpack.includes>
```

Comment:
Given that KFS will no longer depend on rice do we still need this section of the pom file for unpacking dependencies that was added to fix the build process after moving to Tomcat 8.5? Is this something that was going to be further changed as part the the next user story? Thank you.

Reply:
We originally needed this unpacking for the Java 11 cutover, to force some of the Rice files (or Cynergy-customized Rice files) to take precedence. I'm not sure if the "META-INF" unpacking is still needed, so I will look into cleaning that up further in the follow-up user story.

We also needed to update this unpacking as part of the KIM-to-KFS migration. I can't remember exactly, but I think we were running into issues where some customized KIM Java classes were in the same WAR directory as some of the KIM XML files, and I believe we had to forcibly unpack them so that the resource-loading code could find the XML files properly. We might still need the KIM-unpacking section as a result.

Reply:
Thank you for explaining this. I did not remember we also needed this for the KIM-to-KFS migration.
Further cleaning this up as part of the follow up user story sounds good.